src/data/queries/healthCareLocalFacility.ts

```diff
@@ -88,6 +91,20 @@ export const formatter: QueryFormatter<
   const formattedMenu =
     menu !== null ? buildSideNavDataFromMenu(entity.path.alias, menu) : null
 
+  // Sort the health care services by name
```

Comment:
If it doesn't have a description, put it first?

Reply:
🤔 It should be the other way around.
```ts
    if (nameA === undefined && nameB === undefined) return 0
    if (nameA === undefined) return 1 // a goes after b
    if (nameB === undefined) return -1 // a goes before b
```

```
If neither has a name, it doesn't matter
If the first has no name, the second goes first
If the first _does_ have a name, but the second doesn't, the first goes first
```